README.md

```diff
@@ -2,7 +2,7 @@
 
 This is the Home of the Jitsuin RKVST Documentation
```

Comment:
Do we need to change this to just RKVST as well or is this okay on it's own?

Reply:
I've just found a bunch of this in the metadata and so on ... will update